---
title: AsposeArgumentOutOfRangeException
second_title: Aspose.Email für .NET-API-Referenz
description: Initialisiert eine neue Instanz vonAsposeArgumentOutOfRangeExceptionaspose.email/asposeargumentoutofrangeexception Klasse.
type: docs
weight: 10
url: /de/net/aspose.email/asposeargumentoutofrangeexception/asposeargumentoutofrangeexception/
---
## AsposeArgumentOutOfRangeException() {#constructor}

Initialisiert eine neue Instanz von[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception) Klasse.

```csharp
public AsposeArgumentOutOfRangeException()
```

### Siehe auch

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* namensraum [Aspose.Email](../../asposeargumentoutofrangeexception)
* Montage [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string) {#constructor_2}

Initialisiert eine neue Instanz von[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception) Klasse mit einer bestimmten Fehlermeldung.

```csharp
public AsposeArgumentOutOfRangeException(string message)
```

| Parameter | Typ | Beschreibung |
| --- | --- | --- |
| message | String | Die Nachricht, die den Fehler beschreibt. |

### Siehe auch

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* namensraum [Aspose.Email](../../asposeargumentoutofrangeexception)
* Montage [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string, params object[]) {#constructor_5}

Initialisiert eine neue Instanz von[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception) Klasse mit einer bestimmten Fehlermeldung.

```csharp
public AsposeArgumentOutOfRangeException(string format, params object[] parameters)
```

| Parameter | Typ | Beschreibung |
| --- | --- | --- |
| format | String | Nachrichtenformat |
| parameters | Object[] | Parameter formatieren |

### Siehe auch

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* namensraum [Aspose.Email](../../asposeargumentoutofrangeexception)
* Montage [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string, int) {#constructor_3}

```csharp
public AsposeArgumentOutOfRangeException(string format, int parameter)
```

### Siehe auch

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* namensraum [Aspose.Email](../../asposeargumentoutofrangeexception)
* Montage [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string, Exception) {#constructor_4}

Initialisiert eine neue Instanz von[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception) Klasse mit einer angegebenen Fehlermeldung und einem Verweis auf die innere Ausnahme, die diese Ausnahme verursacht hat.

```csharp
public AsposeArgumentOutOfRangeException(string message, Exception innerException)
```

| Parameter | Typ | Beschreibung |
| --- | --- | --- |
| message | String | Die Fehlermeldung, die den Grund für die Ausnahme erklärt. |
| innerException | Exception | Die Ausnahme, die die Ursache der aktuellen Ausnahme ist, oder ein Nullverweis (Nothing in Visual Basic), wenn keine innere Ausnahme angegeben ist. |

### Siehe auch

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* namensraum [Aspose.Email](../../asposeargumentoutofrangeexception)
* Montage [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string, string) {#constructor_6}

```csharp
public AsposeArgumentOutOfRangeException(string message, string paramName)
```

| Parameter | Typ | Beschreibung |
| --- | --- | --- |
| message | String |  |
| paramName | String |  |

### Siehe auch

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* namensraum [Aspose.Email](../../asposeargumentoutofrangeexception)
* Montage [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(Exception) {#constructor_1}

Initialisiert eine neue Instanz von[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception) Klasse mit einem Verweis auf die innere Ausnahme, die die Ursache dieser Ausnahme ist.

```csharp
public AsposeArgumentOutOfRangeException(Exception innerException)
```

| Parameter | Typ | Beschreibung |
| --- | --- | --- |
| innerException | Exception | Die Ausnahme, die die Ursache der aktuellen Ausnahme ist, oder ein Nullverweis (Nothing in Visual Basic), wenn keine innere Ausnahme angegeben ist. |

### Siehe auch

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* namensraum [Aspose.Email](../../asposeargumentoutofrangeexception)
* Montage [Aspose.Email](../../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.Email.dll -->
